### PR TITLE
feat: attestation everywhere

### PR DESCRIPTION
**Pull request by jleach**
Bring in the Bifold changes that allow attestation to work form anywhere in the application.

### CODE REVIEW

**Comment by bryce-mcmath** [comment]
@jleach some sonarcloud issues

File: app/ios/Podfile.lock
Comment (bryce-mcmath) [inline app/ios/Podfile.lock:1]: Does this work on iOS? It looks like you've removed the attestation pod and haven't replaced it with anything?

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Question about Podfile

File: app/ios/Podfile.lock
Comment (jleach) [reply app/ios/Podfile.lock:1]: I must be missing something in the attestation controller that is causing it to not be picked up. Looking.

Comment (jleach) [reply app/ios/Podfile.lock:1]: Found it. Missing `react-native-attestation.podspec`.

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**